SampleApplications/SDK/Opc.Ua.Client/Session.cs

```diff
@@ -1644,6 +1644,20 @@ public Node ReadNode(NodeId nodeId)
                             continue;
                         }
 
+                        // check for optional attributes for which access denied can be ignored
+                        if (values[ii].StatusCode == StatusCodes.BadUserAccessDenied)
```

Comment:
maybe all Bad StatusCodes can be ignored for optional attributes...
   // check for errors for optional attributes
   if ( StatusCode.IsBad(values[ii].StatusCode))

Reply:
e.g. some servers may return BadSecurityModeInsufficient

Reply:
yeah it makes sense, I fixed it, Thanks!